.git-blame-ignore-revs

```diff
@@ -0,0 +1,2 @@
+# Add `.pre-commit-config.yaml` + `pre-commit run -a`
+9fada6b65f633ecbbfb8a5ef8e36e0c4fc573266
```

Comment:
```suggestion
```

Reply:
WDYM? Remove the sha1 from the file AKA "not-ignore" its blame?